README.md

```diff
@@ -39,9 +39,7 @@ $GLOBALS['TYPO3_CONF_VARS']['LOG']['writerConfiguration'][\TYPO3\CMS\Core\Log\Lo
 To get additional information for each error/warning logged, add the breadcrumb writer as well
 
 ```php
-$GLOBALS['TYPO3_CONF_VARS']['LOG']['writerConfiguration'][\TYPO3\CMS\Core\Log\LogLevel::WARNING] = [
-    \Helhum\SentryTypo3\Log\Writer\SentryBreadcrumbWriter::class => [],
-];
+$GLOBALS['TYPO3_CONF_VARS']['LOG']['writerConfiguration'][\TYPO3\CMS\Core\Log\LogLevel::WARNING][\Helhum\SentryTypo3\Log\Writer\SentryBreadcrumbWriter::class] = [];
```

Comment:
Can you make this `LogLevel::INFO`? This makes more sense, because otherwise breadcrumbs do not add more context to the log sentry entry

Reply:
I'll do this myself then